zone/jt

```diff
@@ -0,0 +1,2 @@
+jt    IN      A       2.51.170.185:1234 (Port 80 cannot be opened on my server)
+jt    IN      NS      91.217.137.37
```

Comment:
```suggestion
```
NS records typically point to nameservers, and they have to be hostnames, not IP addresses. If you don't have your own nameserver in operation you can remove this line. Also you can't mix NS and A records. See: http://www.zytrax.com/books/dns/ch8/ns.html